products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+    latestReleaseDate: 2016-05-18
+    releaseDate: 2016-05-18
+
+-   releaseCycle: "3.12"
+    eol: false
+    latest: "3.12.2"
+    latestReleaseDate: 2016-04-18
+    releaseDate: 2016-03-29
+
+-   releaseCycle: "3.11"
+    eol: false
+    latest: "3.11.1"
+    latestReleaseDate: 2016-03-03
+    releaseDate: 2016-02-15
+
+-   releaseCycle: "3.10"
+    eol: false
+    latest: "3.10.2"
+    latestReleaseDate: 2016-01-20
+    releaseDate: 2016-01-06
+
+-   releaseCycle: "3.09"
+    eol: false
+    latest: "3.09.02"
+    latestReleaseDate: 2015-11-02
+    releaseDate: 2015-10-14
+
+-   releaseCycle: "3.08"
+    eol: false
+    latest: "3.08.11.01"
+    latestReleaseDate: 2015-07-29
+    releaseDate: 2013-08-26
+
+-   releaseCycle: "3.07"
+    eol: false
+    latest: "3.07.17"
+    latestReleaseDate: 2013-05-20
+    releaseDate: 2010-07-21
+
+-   releaseCycle: "3.06"
+    eol: false
+    latest: "3.06.23.01"
+    latestReleaseDate: 2010-03-26
+    releaseDate: 2008-07-16
+
+-   releaseCycle: "3.05"
+    eol: false
+    latest: "3.05.09"
+    latestReleaseDate: 2008-05-14
+    releaseDate: 2007-09-04
+
+-   releaseCycle: "3.04"
+    eol: false
+    latest: "3.04.02"
+    latestReleaseDate: 2007-08-13
+    releaseDate: 2007-06-18
+
+-   releaseCycle: "3.03"
+    eol: false
+    latest: "3.03.17"
+    latestReleaseDate: 2007-04-25
+    releaseDate: 2006-01-11
+
+-   releaseCycle: "3.02"
+    eol: false
+    latest: "3.02.08"
+    latestReleaseDate: 2005-12-19
+    releaseDate: 2005-03-21
+
+-   releaseCycle: "3.01"
+    eol: false
+    latest: "3.01.06"
+    latestReleaseDate: 2005-03-17
+    releaseDate: 2005-01-21
+
+-   releaseCycle: "3.0"
+    eol: false
+    latest: "3.0.8"
+    latestReleaseDate: 2004-10-12
+    releaseDate: 2004-06-18
+
+-   releaseCycle: "2.08"
+    eol: false
+    latest: "2.8.14"
+    latestReleaseDate: 2004-06-09
+    releaseDate: 2003-02-16
+
+-   releaseCycle: "2.07"
+    eol: false
+    latest: "2.7.6"
+    latestReleaseDate: 2003-01-25
+    releaseDate: 2002-08-25
+
+-   releaseCycle: "2.06"
+    eol: false
+    latest: "2.6.3"
+    latestReleaseDate: 2002-08-13
+    releaseDate: 2002-07-18
+
+-   releaseCycle: "2.05"
+    eol: false
+    latest: "2.5.6"
+    latestReleaseDate: 2002-07-07
+    releaseDate: 2002-06-17
+
+-   releaseCycle: "2.04"
+    eol: false
+    latest: "2.04.12"
+    latestReleaseDate: 2002-05-10
+    releaseDate: 2002-03-11
+
+-   releaseCycle: "2.03"
+    eol: false
+    latest: "2.03.03"
+    latestReleaseDate: 2002-02-19
+    releaseDate: 2002-02-03
+
+-   releaseCycle: "2.02"
+    eol: false
+    latest: "2.02.05"
+    latestReleaseDate: 2002-01-28
+    releaseDate: 2001-12-22
+
+-   releaseCycle: "2.01"
+    eol: false
+    latest: "2.01.07"
+    latestReleaseDate: 2001-12-15
+    releaseDate: 2001-11-12
+
+-   releaseCycle: "2.0"
+    eol: false
+    latest: "2.00.08"
+    latestReleaseDate: 2001-11-04
+    releaseDate: 2001-09-28
+
+-   releaseCycle: "1.0"
+    eol: false
```

Comment:
Is 1.x version, with latest on 2001-07-23, still supported ?

The same comment applies to all cycles.

Reply:
According to their [Long Term Support ](https://www.sqlite.org/lts.html) it looks like they all are supported : do you share the same analysis ?

Reply:
I do not have the same opinion about the LTS policies. I they are just saying they plan to develop SQLite as if the project will be supported until 2050.

You should be looking at https://sqlite.org/com/tech.html :

>  We will support any version of SQLite up to a year old, **not just the latest version**. At your request, we will backport any bug fixes to any SQLite version that is less than one year old. 

Given that information I also think that the cycles must be the `major` version, not `major.minor`. It will be easier to maintain.

Reply:
Versions collapsed and description updated.